### ISSUE TITLE
time-sensitive bucharest bug

### ISSUE THREAD

**Issue by spencermountain** [spencermountain/spacetime#80]
set computer time to 7:30pm EST

```js
console.log(spacetime('March 1st 2015', 'Europe/Bucharest').format('nice'))
//Mar 31st, 2:43am
```

only march overflows. error begins at `7:20pm` and ends at 8:20pm.

bucharest has a dst switch at `03/25:03`, may be related

**Comment by spencermountain** [comment]
issue here:
https://github.com/spencermountain/spacetime/blob/cdd5ba189d70d3d42e8ec39c3e38093bae094bad/src/methods/set/walk.js#L24

**Comment by spencermountain** [comment]
fixed in 5.1.0
